api/comments_api.py

```diff
@@ -72,18 +80,24 @@ def do_get(self, **kwargs) -> dict[str, list[dict[str, Any]]]:
       lambda c: self._should_show_comment(c, user_email, is_admin), comments))
 
     dicts = [activity_to_json_dict(c) for c in comments]
-    return {'comments': dicts}
+    return GetCommentsResponse(comments=dicts).to_dict()
 
   def do_post(self, **kwargs) -> dict[str, str]:
     """Add a review comment and possibly set a approval value."""
     feature_id = kwargs['feature_id']
     gate_id = kwargs.get('gate_id', None)
     feature = self.get_specified_feature(feature_id=feature_id)
     user = self.get_current_user(required=True)
-    post_to_thread_type = self.get_param(
-        'postToThreadType', required=False)
 
-    comment_content = self.get_param('comment', required=False)
+    json_data = self.request.json
+    comment_request_data = {
+        'comment': json_data['comment'],
+        'post_to_thread_type': json_data.get('postToThreadType', None),
+    }
+    comment_request = CommentsRequest(**comment_request_data)
```

Comment:
I looked into the generated controller and model classes and it looks like the `from_dict()` method is the easiest way to have the generated code validate the input.   So, could these 6 lines be just one line?
`comment_request = CommentRequest.from_dict(self.request.json)`

Reply:
this works and is a lot easier! Thanks